content/blog/the-rise-of-special-project-infra.md

```diff
@@ -0,0 +1,168 @@
+      | x264    | FHD  | 1920x1080        | 1900-4500 kbps | 25 or 30 fps |
+
+      Note: some services recommend a higher bitrate for HD:
+
+      | Encoder | Name | Resolution (WxH) |   Bitrate | Framerate    |
+      | ------- | ---- | ---------------- | --------: | ------------ |
+      | x264    | HD   | 1280x720         | 3000 kbps | 25 or 30 fps |
+      | x264    | FHD  | 1920x1080        | 4500 kbps | 25 or 30 fps |
+
+      Note: our stress tests showed that the use of standard variants with our VPS were reportedly adequate for most users but problematic for some in Japan, Thailand, and the UK. Based on these results, object storage and CDN caching are recommended.
+
+      Additional notes:
+
+      - For interactive live streams (e.g., lectures with Q&A sessions), consider decreasing the latency buffer to Low.
+      - For further tuning, consider referring to a bitrate calculator (e.g., [Bitrate Calc](https://bitratecalc.com/)).
+
+#### Streaming client configuration
+
+With the server side all set, it was time to get streaming! The basic procedure follows:
+
+1. On a local workstation, install and launch [OBS Studio](https://obsproject.com/) ([obs-studio](https://search.nixos.org/packages?channel=22.05&show=obs-studio&from=0&size=50&sort=relevance&type=options&query=obs-studio) in Nixpkgs).
+
+1. In stream settings, set the following values:
+
+   - Service: `Custom...`
+   - Server: `rtmp://<target FQDN>` (e.g., `rtmp://live.nixos.org`)
+   - Stream key: `abc123` (by default)
+
+1. Go live.
+
+1. Confirm the stream at the designated URL (`https://live.nixos.org` in our case).
+
+_It worked?_ It worked. Crisis averted with time to spare. 🍹
+
+## Next Steps
+
+Following a successful first lecture, the SoN infra team is concerned not only with maintenance and optimization of the infrastructure we’ve already deployed but also with innovation on a larger scale. The ultimate goal is to contribute a fully self-hosted, documented, NixOS-based, general-purpose, FLOSS[^floss] conference suite that not only is usable for the SoN Public Lecture Series but also can scale for a modern conference experience—namely [NixCon](https://nixcon.org/).
+
+In addition to main conference components (e.g., administrative tools, breakout rooms, etc.) and general convenience features (e.g., collaboration tools such as shared whiteboards), a notable priority is the improvement of our accessibility story, e.g., through integration of the [Vosk speech recognition toolkit](https://alphacephei.com/vosk/) for livestream closed captioning/transcription/subtitling (in multiple languages).
+
+To this end, we’re working together with other great teams such as the SoN Special Project for [Jitsi](https://jitsi.org/) and have received guidance and insight from the gracious maintainers of the [NixCon 2020 livestreaming infrastructure](https://github.com/nixcon/nixcon-video-infra)—the crucial project of which our efforts will be a continuation.
```

Comment:
```suggestion
To this end, we’re working together with other teams such as the SoN Special Project for [Jitsi](https://jitsi.org/) and have received guidance and insight from the gracious maintainers of the [NixCon 2020 livestreaming infrastructure](https://github.com/nixcon/nixcon-video-infra)—the crucial project of which our efforts will be a continuation.
```

I try to abstain from value judgements for the sake of clarity, unless you would want to explain what makes anything "great".

Reply:
While I wholeheartedly agree in the context of technical writing, this blog post is in its current form more of an opinion piece styled to spark interest in SoN, the Nix Ecosystem, and self-hosting with Free Software – and as such is brimming with value judgements. Granted the technical density of the instructional content caters to the target readership, documentation-grade objectivity wasn’t a general objective.

Ultimately, I’m trying to understand the scope of this suggestion. I certainly don’t mind taking out simple modifiers like “great”, but unless I’m reading too far into the suggestion, the removal of embellishments like this could be a slippery slope.

Reply:
It's clear from your style that this is an entertainment and opinion piece sprinkled with technical instructions. I'm always fine with mixing in opinion and value judgement in this context if it it's at least minimally qualified. Just being great does not tell much.

Reply:
👍